Configure.pl

```diff
@@ -17,6 +17,11 @@
 sub defined_or($$) {
     defined $_[0] ? $_[0] : $_[1]
 }
+# This allows us to run on broken perls
```

Comment:
What does broken mean in this context?

Reply:
Does not ship `List::Util`, i.e shipped by redhat and friends.